paddle/fluid/operators/cinn/cinn_launch_op.h

```diff
@@ -134,7 +135,14 @@ class CinnLaunchOpKernel : public framework::OpKernel<T> {
 
     // Step 4. Execute the compiled CINN instructions by a PE or
     //         by the CINN compiled program in sequential order
-    if (FLAGS_enable_pe_launch_cinn) {
+    if (FLAGS_enable_interpretercore_launch_cinn) {
+      platform::RecordEvent record_event_4(
+          "Step 4. Execute the runtime program by InterpreterCore.");
+      VLOG(4) << "Execute the runtime program by InterpreterCore";
+      auto* interpreter_core = launch_context->InitializeInterpreterCore(
+          place, const_cast<framework::Scope*>(&scope));
+      interpreter_core->Run({});
+    } else if (FLAGS_enable_pe_launch_cinn) {
```

Comment:
```suggestion
    if (FLAGS_enable_pe_launch_cinn) {
         if (FLAGS_enable_interpretercore_launch_cinn) {
           platform::RecordEvent record_event_4(
           "Step 4. Execute the runtime program by InterpreterCore.");
      VLOG(4) << "Execute the runtime program by InterpreterCore";
      auto* interpreter_core = launch_context->InitializeInterpreterCore(
          place, const_cast<framework::Scope*>(&scope));
      interpreter_core->Run({});
      } else {
         ... pe->RunWithoutFetch
      }
    }  else {
       ... LaunchCinnExecution
    }
```
建议这个开关放到`FLAGS_enable_pe_launch_cinn` if 分支内，因为当前有些开发是通过关闭FLAGS_enable_pe_launch_cinn开关走LaunchCinnExecution执行方式进行测试，此处逻辑会导致不兼容

Reply:
Done, tks!